dm/syncer/syncer.go

```diff
@@ -972,16 +972,21 @@ func (s *Syncer) addJob(job *job) {
 }
 
 // checkShouldFlush checks whether syncer should flush now because last flushing is outdated.
-func (s *Syncer) checkShouldFlush() {
+func (s *Syncer) checkShouldFlush() error {
 	if !s.checkpoint.CheckGlobalPoint() || !s.checkpoint.CheckLastSnapshotCreationTime() {
-		return
+		return nil
 	}
 
 	jobSeq := s.getFlushSeq()
 	s.tctx.L().Info("Start to async flush current checkpoint to downstream based on flush interval", zap.Int64("job sequence", jobSeq))
-	j := newAsyncFlushJob(s.cfg.WorkerCount, jobSeq)
-	s.addJob(j)
-	s.flushCheckPointsAsync(j)
+	if s.cfg.Experimental.AsyncCheckpointFlush {
```

Comment:
we need to change the log info "Start to async flush current checkpoint..."

Reply:
```
jobSeq := s.getFlushSeq() 
```
Only needed in async mode

Reply:
done.